backend/utils/utils.py

```diff
@@ -41,7 +41,7 @@ def get_similar_verses(user_input):
     similarities = cosine_similarity(user_input_vector, cluster_vectors).flatten()
 
     # Get the indices of the top 5 most similar verses
-    top_indices = similarities.argsort()[-5:][::-1]
+    top_indices = similarities.argsort()[-20:][::-1]
```

Comment:
Why is this set to 20? Why not fetch all??

Reply:
I was initially instructed to limit the results to a maximum of 20  However, if you'd prefer to fetch all results, I'm happy to adjust the implementation accordingly.

Reply:
Yeah! Do update it to fetch all

Reply:
Wait @Namit2111 he is only supposed to do 20 because we don’t want it to be an endless list and have good percentages. 20 is enough because we have no way of collapsing that list back to 5 after clicking let’s say 10 times (50 shown)

Reply:
Then should  we try pagination?

Reply:
We could but thought the see more button would have been an easier start for this type of feature. We could merge this and then segway into pagination. Just unsure how hard that would be to do. @Namit2111 

If we don’t touch the routing and rather do a slide of “pages” or windows of 5 I can’t imagine it being that hard to display results, will take some refactoring though

Reply:
sure let's merge this first then , we will look into pagination later on